ethereum/dex/trades.sql

```diff
@@ -655,6 +655,27 @@ WITH rows AS (
 
         UNION ALL
 
+        -- 1inch LP
+        SELECT
+            evt_block_time,
+            '1inch LP' AS project,
+            '1' AS version,
+            'DEX' AS category,
+            sender AS trader_a,
+            NULL::bytea AS trader_b,
+            result AS token_a_amount_raw,
+            amount AS token_b_amount_raw,
+            NULL::numeric AS usd_amount,
+            "dstToken" AS token_a_address,
```

Comment:
One of `dstToken` or `srcToken` seems to be always `0`. Why is that? Maybe you need to preprocess the events somehow to conform them to this table? The goal is that `token_a` is the token the trader is buying and `token_b` is the token the trader is selling to the pool.

Reply:
Oh I see, the pools hold ETH, not WETH

Reply:
In this case, I would suggest an analogous approach to what is done for eg Uniswap v1 and Kyber in this table: replace `0` with `\xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2 ` (WETH)

Reply:
`0` means ETH as well as `0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee`

Reply:
Yea, implementing the same `CASE` that was done for Bancor should be enough to make sure the 1inch data is properly accounted for in `dex.trades` and all the queries and dashboards that use it and expect WETH in the `token_address` fields.

Reply:
Fixed for both 1LP and Mooniswap